LoopKitUI/CGMManagerUI.swift

```diff
@@ -33,7 +33,7 @@ public protocol CGMStatusIndicator {
     func glucoseRangeCategory(for glucose: GlucoseSampleValue) -> GlucoseRangeCategory?
 }
 
-public protocol CGMManagerUI: CGMManager, DeviceManagerUI, PreferredGlucoseUnitObserver, CGMStatusIndicator {
+public protocol CGMManagerUI: CGMManager, DeviceManagerUI, DisplayGlucoseUnitObserver, CGMStatusIndicator {
```

Comment:
Do we need this now?

Reply:
`DexcomCGMManager` still does (or something similar). Its alert manager submits notifications with display glucose units. 

Alternatively, the CGMManager protocol could include `DisplayGlucoseUnitObservable` property.

Reply:
Ah, okay, I see. Yea, not sure of the right answer here, but we can leave it as-is.